app/processing/account-codes/maps/cs.js

```diff
@@ -0,0 +1,76 @@
+const { SOS710, SOS920, SOS921, SOS922, SOS923, SOS924, SOS925, SOS926, SOS927, SOS928 } = require('../../../constants/account-codes/ap')
+const { SOS750, SOS940, SOS941, SOS942, SOS943, SOS944, SOS945, SOS946, SOS948 } = require('../../../constants/account-codes/ar-admin')
+const { SOS770, SOS960, SOS961, SOS962, SOS963, SOS964, SOS965, SOS966, SOS967, SOS968 } = require('../../../constants/account-codes/ar-irregular')
```

Comment:
See my comment here: https://github.com/DEFRA/ffc-pay-processing/pull/121/files#r1192436374

Reply:
As before I put them in the order being used, but this one is a bit different as the last code uses an existing one so doesn't quite work.

What do you think?  Either way I'm happy

Reply:
Happy to leave this as is